docs/upgrading.rst

```diff
@@ -0,0 +1,156 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Upgrading to version 3.2
+========================
+
+If you have your instance of Invenio v3.1 already up and running and
+you would like to upgrade to version v3.2 you don't need to set up your
+project from scratch. The goal of this guide is to show the steps to upgrade
+your project without losing any of your work.
+
+
+Pipfile modifications
+---------------------
+
+The most important changes that you will have to make are in ``Pipfile``.
+
+First you need to change the invenio version:
+
+::
+
+    invenio = { version = "==3.2.0a7", extras = ["base", "auth", "metadata", "files", "postgresql", "elasticsearch7" ]}
+
+If you want to use the new files bundle make sure you include the ``files``
+bundle. Add any additional :ref:`bundles` you would like in your project in
+``extras``.
+
+To install the new packages in ``Pipfile`` run the following commands:
```

Comment:
Please discuss this section with someone, in general it should not contain --dev unless you say somewhere before it is about upgrading first on the dev environment.

Reply:
Actually `--dev` should not be needed for upgrading a production system.